src/Compilers/Test/Resources/Core/project.json

```diff
@@ -1,7 +1,9 @@
 {
   "supports": { },
-  "dependencies": { },
+  "dependencies": {
+    "System.Xml.ReaderWriter": "4.0.11"
```

Comment:
Need reference to `NETStandard.Library` and `Microsoft.NETCore.Portable.Compatibility`